packages/api/src/mappings/fitbit/body.ts

```diff
@@ -28,35 +28,37 @@ export const mapToBody = (
     metadata: metadata,
   };
 
-  if (fitbitUser.user.height) {
-    if (fitbitUser.user.heightUnit === US_LOCALE) {
-      body.height_cm = parseFloat(fitbitUser.user.height.toFixed(DECIMAL_PLACES));
-    } else {
-      body.height_cm = parseFloat(
-        convert(fitbitUser.user.height).from("in").to("cm").toFixed(DECIMAL_PLACES)
-      );
+  if (fitbitUser) {
+    if (fitbitUser.user.height) {
+      if (fitbitUser.user.heightUnit === US_LOCALE) {
+        body.height_cm = parseFloat(fitbitUser.user.height.toFixed(DECIMAL_PLACES));
+      } else {
+        body.height_cm = parseFloat(
+          convert(fitbitUser.user.height).from("in").to("cm").toFixed(DECIMAL_PLACES)
+        );
+      }
```

Comment:
I know it was already like this, but its odd that if the locale is US we don't do conversion to centimeters... shouldn't it be the other way around (US uses imperial and thus needs conversion to metric system)?

Reply:
Good catch ill fix